pkg/tarmak/tarmak.go

```diff
@@ -365,3 +374,135 @@ func (t *Tarmak) CmdKubectl(args []string) error {
 func (t *Tarmak) CancellationContext() interfaces.CancellationContext {
 	return t.ctx
 }
+
+func (t *Tarmak) DestroyEnvironment() error {
+	inputDestroy := input.New(os.Stdin, os.Stdout)
+
+	destroyClusters := true
```

Comment:
Perhaps create a `var err error` here so we don't have to create lots of response bool variables, i.e
```
var err error
destroy := true
...
destroy, err = inputDestroy(....
...
if destroy {...
.....
```